### PR TITLE
chore(ui5-message-strip): migrate wdio to cy

### PR DESCRIPTION
**Pull request by MapTo0**

### CODE REVIEW

File: packages/main/cypress/specs/MessageStrip.cy.ts [hunk 1]
```diff
@@ -1,8 +1,92 @@
...
+			$strip.get(0).addEventListener("close", cy.stub().as("close"));
+		});
+
+		cy.get("[ui5-message-strip]")
+			.shadow()
+			.find(".ui5-message-strip-close-button")
+			.as("closeButton");
+
+		cy.get("@closeButton")
+			.click();
```
Comment (nnaydenow) [inline packages/main/cypress/specs/MessageStrip.cy.ts:25]: realClick

File: packages/main/cypress/specs/MessageStrip.cy.ts [hunk 2]
```diff
@@ -1,8 +1,92 @@
...
+		// Arrange
+		cy.mount(html`
+			<div>
+					<ui5-message-strip id="colorSet1ColorScheme1" design="ColorSet1">MessageStrip 1</ui5-message-strip>
+					<ui5-message-strip id="colorSet1ColorScheme2" design="ColorSet2">MessageStrip 2</ui5-message-strip>
+			</div>
+		`);
+
+		// Act & Assert
+		cy.get("#colorSet1ColorScheme1").should("have.attr", "design", "ColorSet1");
```
Comment (nnaydenow) [inline packages/main/cypress/specs/MessageStrip.cy.ts:46]: this test is testing framework logic. probably it is redundant.

File: packages/main/cypress/specs/MessageStrip.cy.ts [hunk 3]
```diff
@@ -1,8 +1,92 @@
...
+		<ui5-message-strip id="colorScheme7" design="ColorSet1" color-scheme="7" class="top"><ui5-icon name="palette" slot="icon"></ui5-icon>Color Set 1 - color-scheme 7</ui5-message-strip>
+		`);
+
+		// Arrange
+		cy.get("#defaultColorScheme").as("messageStripWithoutScheme");
+		cy.get("#colorScheme7").as("messageStripColorScheme");
+
+		// Assert
+		cy.get("@messageStripWithoutScheme").should("have.prop", "colorScheme", "1");
+		cy.get("@messageStripColorScheme").should("have.prop", "colorScheme", "7");
```
Comment (nnaydenow) [inline packages/main/cypress/specs/MessageStrip.cy.ts:68]: same here

File: packages/main/cypress/specs/MessageStrip.cy.ts (hunk 2 above)
Comment (MapTo0) [reply packages/main/cypress/specs/MessageStrip.cy.ts]: yeah, this functionality is mostly colors, visual tests would be a nice replacement :)

**Review by ilhan007 — APPROVED** [review]

**Pull request merged**